### PR TITLE
Added a Google Maps to portfolio page

### PR DESCRIPTION
**Pull request by colemanliyah**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -30,6 +30,7 @@
 
 @WebServlet("/comments")
 public class DataServlet extends HttpServlet {
+    private int maxComments = 5;
```
Comment (ofk-goog) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: This change is out of scope for this pull request

File: portfolio/src/main/java/com/google/sps/servlets/RequestServlet.java [hunk 2]
```diff
@@ -13,26 +13,34 @@
 import javax.servlet.http.HttpServletResponse;
```
Comment (ofk-goog) [inline portfolio/src/main/java/com/google/sps/servlets/RequestServlet.java:13]: I don't think the changes to this file are related to the new map functionality

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -31,12 +31,6 @@ function getComments(){
     });
 }
 
-function comments_to_display(){
-    let num = document.getElementById("num_of_comments");
-    let number = num.options[num.selectedIndex].value;
-    console.log(number);
-}
-
```
Comment (ofk-goog) [inline portfolio/src/main/webapp/script.js:39]: Deleting this seems unrelated to the map. Ditto the function `addRandomGreeting`. If these functions were unused, I guess it's fine to delete them in this PR. But please mention in the PR description that you're also deleting some unused code.

**Review by ofk-goog — CHANGES_REQUESTED** [review]
The new map stuff looks good. There are still some things in this PR that look unrelated, though

File: portfolio/src/main/java/com/google/sps/servlets/RequestServlet.java (hunk 2 above)
Comment (colemanliyah) [reply portfolio/src/main/java/com/google/sps/servlets/RequestServlet.java:13]: That is actually pretty wired that this shows up as an added change...I've always had this import here because the doGet and doPost functions don't work without this

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (colemanliyah) [reply portfolio/src/main/webapp/script.js:39]: Okay, these were functions I just didn't need anymore. I'll be sure to make note of them next time

**Review by ofk-goog — APPROVED** [review]
